index.html

```diff
@@ -19,7 +19,14 @@ <h1 id="title">Survey Form</h1>
         <label for="email" id="email-label">Email</label>
         <input type="email" id="email" placeholder="Your Email" required />
         <label for="number" id="number-label">Tel</label>
-        <input type="number" id="number" min="10" max="15" placeholder="Number" required />
+        <input
+          type="number"
+          id="number"
```

Comment:
- [ ] Please visit [this link to](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate) learn more about how to communicate while giving code review comments.